sympy/solvers/inequalities.py

```diff
@@ -1004,3 +1004,115 @@ def reduce_inequalities(inequalities, symbols=[]):
 
     # restore original symbols and return
     return rv.xreplace({v: k for k, v in recast.items()})
+
+def _fourier_motzkin_extension(inequalities):
+    """
+    Extension of the Fourier-Motzkin algorithm to the case where inequalities does not contains variables that have at
+    least two coefficients with opposite sign.
+
+    Examples
+    ========
+    >>> from sympy.solvers.inequalities import _fourier_motzkin_extension
```

Comment:
empty line after `======` and import things needed (e.g., `>>> from sympy.abc import x, y, z`)